### ISSUE TITLE
Attention: kwirkykat 

### ISSUE THREAD

**Issue by forensicsguy20012004** [PowerShell/xSQLServer#78]
Kwirkykat,

You are doing a great job with this repository...with that said, I was wondering if you could do all of us a favor and produce some documentation, at regular interval, containing:

1. the current state of the master repository [the read-me is extremely long..just need points]
2. state of open issues and open pull requests and what the pull/merge requests include

Why you ask? I saw that Troy Ault was in the process of getting his stuff into the master...as it took care of my request for SQL 2016 and some other changes...not sure where it all stands. I wanted to download the newest repository with the newest code...but saw it did not include Troy's stuff. 

Basically, kind of looking for a forecasting...indicating what will be merged and what that merge includes. Essentially, if we were doing "sprints" what our sprint duration is and what is included in that sprint. Some "low-level" management reports for the repository.

It would be great, if not I understand we all know you are super busy. Keep up the good work, we all really appreciate the time and energy you have put into this repository.

Matt Haines

**Comment by johlju** [comment]
@kwirkykat This one was for you, but since you weren't mentioned you might not have seen this question.

**Comment by kwirkykat** [comment]
@forensicsguy20012004

If you want to download the most recent code that is not yet released, you will need to download the dev branch.
If you want to download code that is not yet merged (e.g. still in a pull request), then you will need to download from the PR author's branch that they are merging from.

The master branch only contains released code (what's on PSGallery).
The dev branch contains all current changes that will be merged into the next release (as long as there are no blocker issues open and all tests are passing).

The README of the dev branch (which yes, is too large) contains an 'Unreleased' section which should contain all the changes to the dev branch that have been merged in that are not yet released/part of the master branch.
This should probably be moved to a Changelog.md like in SharePointDsc.
If there have been PRs merged but the unreleased section has not been updated, then the PR reviewers have not been reviewing their PRs properly and I will need to fix it when the release happens. 

The next release is next week on August 10 which you can see from the [community agenda](https://github.com/PowerShell/DscResources/blob/master/CommunityAgenda.md) in the main DSC Resource Kit repo.

The struggle with this repo specifically is that we no longer have a maintainer for this repository (which we are trying to resolve) so pull requests and issues in this repo are not being monitored and have not been getting their proper labels.
Yes, this is a big problem, and I'm trying to fix it.

**Comment by forensicsguy20012004** [comment]
Thank you so much...I am going to close this. Great Job.
